crates/holochain/src/conductor/conductor/builder.rs

```diff
@@ -433,6 +433,7 @@ impl ConductorBuilder {
                     BASE64_STANDARD.decode(m).map_err(ConductorError::other)
                 })
                 .transpose()?,
+            peer_meta_pruning_interval_ms: Default::default(),
```

Comment:
This `Default::default()` here will default to a value of `0` I think. Isn't that a problem? I'd expect to choose something at the order of 10 seconds to few minutes as a default.

Reply:
I actually *do* think that this will return `0` here because it takes the default of a `u64` and is unrelated to the default of the `HolochainP2pConfig` struct as a whole (I just tested it with some dummy code).

Reply:
Right, I didn't know that. I'll change it.

Reply:
https://github.com/holochain/holochain/pull/5030/commits/7a996b5dae1083f8ee527ffa278adf4f07327500